_posts/events/2024-10-03-METACOL.md

```diff
@@ -0,0 +1,42 @@
+•	Semantics (e.g., degree of lexicalization, motivation/transparency etc.) 
+•	Cognitive Linguistics (e.g., conceptual metaphors and metonymies) 
+•	Construction Grammar (CxG) 
+•	Pragmatics 
+•	Contrastive Linguistics 
+•	(Meta-)Lexicography 
+•	Computational Linguistics and Natural Language Processing (NLP) 
+•	(Machine) Translation 
+•	Language Learning and Teaching 
+•	Language for Specific Purposes (LSP) etc. 
+
+Since the conference is organized within the framework of the scientific research project Metaphorical collocations – syntagmatic relations between semantics and pragmatics (for more information about the project visit the official website https://metakol.uniri.hr/en/), which is financed by the Croatian Science Foundation, the project staff will be presenting their research results pertaining to the project goals. 
+MetaCol’s plenary speakers will include: professor Zoltán Kövecses from the Eötvös Loránd University in Budapest (Hungary), professor Cornelia Müller from the European University Viadrina in Frankfurt/Oder (Germany) and professor Dmitrij Dobrovol’skij, affiliated professor at Stockholm University (Sweden). 
+
+We kindly invite you to participate in the conference by offering your original contributions to the topic. 
+
+The conference will be held both in English and German. Presentations should not exceed 20 minutes and will be followed by a 10-minute slot for discussion. We invite you to submit your abstracts by 31 January 2024 via the following form: https://forms.gle/xbgdSBKfTDc9FRjr9. 
+
+The results of the review process will be announced by 15 April 2024. 
+The conference will be a face-to-face event and no online participation will be possible. Participants are expected to bear the costs of travel and accommodation themselves. The conference fee is 70 EUR for PhD students and 100 EUR for all other participants. It includes conference materials, the conference book, refreshments during breaks and a welcome reception. 
+
+Should you have any questions, please contact the conference organizing committee. 
+
+Kind regards, 
+
+Aneta Stojić (University of Rijeka) aneta.stojic@ffri.uniri.hr 
+Christine Konecny (University of Innsbruck) Christine.Konecny@uibk.ac.at Mihaela Matešić (University of Rijeka) mihaela.matesic@uniri.hr 
+Marija Brkić Bakarić (University of Rijeka) mbrkic@uniri.hr 
+Lucia Načinović Prskalo (University of Rijeka) lnacinovic@inf.uniri.hr 
+Katja Dobrić Basaneže (University Juraj Dobrila in Pula) katja.dobric.basaneze@unipu.hr 
+Jana Jurčević (University of Rijeka) jana.jurcevic@uniri.hr
```

Comment:
If you want these to show up as a vertical list, either make them a bulleted list, or add hard line breaks as per https://github.github.com/gfm/#hard-line-breaks